apps/desktop/src/routes/+page.svelte

```diff
@@ -2,15 +2,16 @@
 	import { ProjectsService } from '$lib/backend/projects';
 	import FullviewLoading from '$lib/components/FullviewLoading.svelte';
 	import { getContext } from '@gitbutler/shared/context';
-	import { derived } from 'svelte/store';
+	import { run } from 'svelte/legacy';
+	import { derived as derivedStore } from 'svelte/store';
```

Comment:
This looks unintentional.

Reply:
No so the migration tool complained about using the store derived in the same file as runes `$derived`. Let me double check though

Reply:
That shouldn't be a problem!